MonoGame.Framework/Utilities/Lz4Stream/Lz4DecoderStream.cs

```diff
@@ -1,13 +1,17 @@
-﻿#define CHECK_ARGS
+﻿// MIT License - Copyright (C) The Mono.Xna Team
```

Comment:
The license and copyright header should not be added as this code came from somewhere else. It wasn't written by us.